holidays/countries/egypt.py

```diff
@@ -42,54 +47,54 @@ def _populate(self, year):
         super()._populate(year)
 
         # New Year's Day
-        self._add_new_years_day("New Year's Day - Bank Holiday")
+        self._add_new_years_day(tr("رأس السنة الميلادية - عطلة بنوك"))
```

Comment:
I'm not sure if it makes sense  to keep "Bank Holiday" here unless it's an official holiday name.

Reply:
It'd be great to address it in a separate PR. For now you can keep it as is or leave a TODO if you'd like.